doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -411,6 +411,11 @@ Other samples
     * A workaround for the hardware `Errata 257`_ of the nRF52811 chip.
     * A workaround for the hardware `Errata 117`_ of the nRF5340 chip.
 
+* :ref:`memfault_sample` sample:
```

Comment:
Shouldn't this be under the 9160 samples, as that is where it used to be and mention there that the sample has been not only moved to another location within NCS, but also moved under Other samples in the documentation structure.

Reply:
I was wondering the same myself. I'll come up with a suggestion to solve both the changelog and Debug samples subpage